docs/UserGuide.md

```diff
@@ -58,8 +71,8 @@ Cautions are placed in this guide to serve as warnings for certain actions.
   * **`delete`**`INDEX` : Deletes the applicant from the list of applicants.
 
   * **`exit`** : Exits the app.
```

Comment:
Indent these bullet points as mentioned in #166, use tab can already inside the markdown file

Reply:
Thanks, fixed!